code/src/pages/NotFound.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+export const NotFound = () => {
+
+  return (
+
```

Comment:
Unsure what best practice is when it comes to empty lines. Personally, I think empty lines helps with readability many times.